crates/motsu/src/shims.rs

```diff
@@ -143,7 +143,7 @@ unsafe extern "C" fn msg_value(value: *mut u8) {
 ///
 /// # Panics
 ///
-/// May panic if fails to parse `CONTRACT_ADDRESS` as an address.
+/// * If fails to parse `CONTRACT_ADDRESS` as an address.
```

Comment:
```suggestion
/// * If it fails to parse `CONTRACT_ADDRESS` as an address.
```
Is this even a correct panic description? In code it panics with `contract_address should be set`

Reply:
replied [here](https://github.com/OpenZeppelin/stylus-test-helpers/pull/34#discussion_r1967483237)